### ISSUE TITLE
Capstone Repo Cloning Failing CI Tests

### ISSUE THREAD

**Issue by andrewstec** [ubccpsc/classy#292]
Issue: Accessibility to a SECapstone organization repository is no longer public, which is causing a CI test to fail. 

Actual: Cloning a SECapstone repository during a CI test fails because of inadequate Git access permissions.

Expected: Cloning a SECapstone repository during a CI test passes because it has adequate permissions to clone the repository. 

![image](https://user-images.githubusercontent.com/7872295/67423694-932c7600-f589-11e9-9a1f-83e8d1a25251.png)
Source: https://circleci.com/gh/ubccpsc/classy/1174

**Comment by andrewstec** [comment]
It turns out that the behaviour of this test was running into a cloning issue and still passing after a long pause until it times out. The cloning issue has been fixed either way.

**Comment by andrewstec** [comment]
Merged PR https://github.com/ubccpsc/classy/pull/294 in 24cfd129a774db3497b56a4406c748075252ab92.
